phpstan.neon

```diff
@@ -0,0 +1,5 @@
+parameters:
+    level: 5
```

Comment:
Level 5 seems to be the lowest that PHPStan will pass for on this repository - did we want to also fix the issues reported at level 6?

Reply:
@tspencer244 You are right. In the near future i would like to achieve level 6. But for now 5 is good, and it is right for the current code.
So: it is ok to have 5.